.vuepress/docs-sidebar.js

```diff
@@ -63,18 +75,38 @@ module.exports = [
     title: 'Configuration Guide',
     collapsable: false,
     children: [
-      ['configuration/', 'Configuration Overview'],
-      'configuration/editors',
-      'configuration/addons',
-      'configuration/things',
-      'configuration/items',
-      'configuration/persistence',
-      'configuration/rules-dsl',
-      ['configuration/blockly/', 'Blockly Rules'],
-      'configuration/transformations',
+      { title: 'Configuration Overview',
```

Comment:
I am not sure whether we want to have this collapsed section or leave it as is - we already spoke about that Stefan.

Reply:
Yep, let's wait for @Confectrician 's feedback on that.

Reply:
Looks good as is.

Reply:
I agree that this needs to be checked.